templates/blob-storage-to-dynamics-365-import.json

```diff
@@ -0,0 +1,379 @@
+							"execute": true,
+							"executionId": "",
+							"legalEntityId": "[LEGAL_ENTITY_HERE]",
+							"overwrite": true,
+							"packageUrl": "@body('Create_SAS_URI_by_path')?['WebUrl']"
+						},
+						"method": "POST",
+						"uri": "[YOUR_D365_INSTANCE_HERE].dynamics.com/data/DataManagementDefinitionGroups/Microsoft.Dynamics.DataEntities.ImportFromPackage"
+					}
+				},
+				"Move_file_to_folder_based_on_execution_success": {
+					"runAfter": {
+						"Wait_For_Execution_Results": [
+							"Succeeded"
+						]
+					},
+					"cases": {
+						"PartiallySuceeded": {
+							"case": "PartiallySucceeded",
+							"actions": {
+								"Copy_file_to_partiallysucceeded_folder": {
+									"runAfter": {},
+									"type": "ApiConnection",
+									"inputs": {
+										"host": {
+											"connection": {
+												"name": "@parameters('$connections')['azureblob']['connectionId']"
+											}
+										},
+										"method": "post",
+										"path": "/datasets/default/copyFile",
+										"queries": {
+											"destination": "/partiallysucceeded/@{body('Copy_file_to_processing_folder')?['Name']}",
+											"overwrite": true,
+											"queryParametersSingleEncoded": true,
+											"source": "@body('Copy_file_to_processing_folder')?['Path']"
+										}
+									}
+								}
+							}
+						},
+						"Succeeded": {
+							"case": "Succeeded",
+							"actions": {
+								"Copy_file_to_succeeded_folder": {
+									"runAfter": {},
+									"type": "ApiConnection",
+									"inputs": {
+										"host": {
+											"connection": {
+												"name": "@parameters('$connections')['azureblob']['connectionId']"
+											}
+										},
+										"method": "post",
+										"path": "/datasets/default/copyFile",
+										"queries": {
+											"destination": "/succeeded/@{body('Copy_file_to_processing_folder')?['Name']}",
+											"overwrite": true,
+											"queryParametersSingleEncoded": true,
+											"source": "@body('Copy_file_to_processing_folder')?['Path']"
+										}
+									}
+								}
+							}
+						}
+					},
+					"default": {
+						"actions": {
+							"Copy_file_to_failed_folder": {
+								"runAfter": {},
+								"type": "ApiConnection",
+								"inputs": {
+									"host": {
+										"connection": {
+											"name": "@parameters('$connections')['azureblob']['connectionId']"
+										}
+									},
+									"method": "post",
+									"path": "/datasets/default/copyFile",
+									"queries": {
+										"destination": "/failed/@{body('Copy_file_to_processing_folder')?['Name']}",
+										"overwrite": true,
+										"queryParametersSingleEncoded": true,
+										"source": "@body('Copy_file_to_processing_folder')?['Path']"
+									}
+								}
+							}
+						}
+					},
+					"expression": "@body('Get_Execution_Results')['value']",
+					"type": "Switch"
+				},
+				"Send_email_if_API_returns_an_error": {
```

Comment:
Consider having this to be failed || skipped || timedout to capture all failure scenarios.